index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Improving the web without third-party cookies</title>
+<title>Third Party Cookies Are Incompatible With Privacy and Must Be Removed</title>
```

Comment:
+1 to @hadleybeeman (https://github.com/w3ctag/web-without-3p-cookies/pull/22#discussion_r1688647375) that

```suggestion
<title>Third Party Cookies Must Be Removed</title>
```

is punchier. (Or even "Third Party Cookies delenda est" ;) But no real objections to the current proposed text.